.github/workflows/cd.yaml

```diff
@@ -0,0 +1,27 @@
+name: cd
+on:
+  workflow_dispatch:
+    inputs:
+      validate_only:
+        required: false
+        type: boolean
+        description: |
+          Run validation with release drafter only
+          → Skip the release job
+        default: true
+  check_run:
+    types:
+      - completed
```

Comment:
What's the use case for the argument and the automatic run on every completed check?

Reply:
> What's the use case for the argument and the automatic run on every completed check?

We run it on each completed check to ensure it's release ready, everything is working and a release could be done on it. 
But still require a manual trigger for a release.

Reply:
> ensure it's release ready, everything is working and a release could be done on it.

This doesn't actually do CI builds, so what is being done here?

Reply:
The release?